### PR TITLE
Issue#114 resource bundle

### PR DESCRIPTION
**Pull request by simonsymhoven**
add external image and font files to internal resources

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by simonsymhoven** [comment]
fixes #115

File: src/main/webapp/css/pull-request-monitoring.css [hunk 1]
```diff
@@ -233,7 +233,7 @@
   font-family: 'Material Icons';
   font-style: normal;
   font-weight: 400;
-  src: url(https://fonts.gstatic.com/s/materialicons/v29/2fcrYFNaTjcS6g4U3t-Y5ZjZjT5FdEJ140U2DJYC3mY.woff2) format('woff2');
+  src: url(../fonts/2fcrYFNaTjcS6g4U3t-Y5ZjZjT5FdEJ140U2DJYC3mY.woff2) format('woff2');
```
Comment (uhafner) [inline src/main/webapp/css/pull-request-monitoring.css:236]: Why are you using a font here? Don't we have the icon ready as a sprite?

File: src/main/resources/modals/config.jelly [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
                     <h4 class="modal-title" id="configModalLabel">
 
-                        <img src="https://www.jenkins.io/images/logos/JCasC/JCasC.png" width="15%" height="15%"/>
+                        <img src="${resURL}/plugin/pull-request-monitoring/icons/JCasC.png" width="15%" height="15%"/>
```
Comment (uhafner) [inline src/main/resources/modals/config.jelly:23]: Wouldn't it make more sense to use the SVG (https://www.jenkins.io/artwork/)?

File: src/main/webapp/css/pull-request-monitoring.css (hunk 1 above)
Comment (simonsymhoven) [reply src/main/webapp/css/pull-request-monitoring.css:236]: Thats the material icons font. What do you mean with 'sprite'?

File: src/main/resources/modals/config.jelly (hunk 2 above)
Comment (simonsymhoven) [reply src/main/resources/modals/config.jelly:23]: Yes thats a good point. I will replace this image and also the plugin icons with svgs.

File: src/main/webapp/css/pull-request-monitoring.css (hunk 1 above)
Comment (uhafner) [reply src/main/webapp/css/pull-request-monitoring.css:236]: I think @fqueiruga included the material icons as sprite in Jenkins core and you can select the corresponding icon by referencing the sprite and the ID within that sprite. 

Here are all supported icons:
https://github.com/jenkinsci/jenkins/tree/master/war/src/main/webapp/images/material-icons

**Review by uhafner — APPROVED** [review]

**Pull request merged**